borsh/Cargo.toml

```diff
@@ -52,6 +57,9 @@ targets = ["x86_64-unknown-linux-gnu"]
 [features]
 default = ["std"]
 derive = ["borsh-derive"]
+async = []
```

Comment:
please rename to `unstable__async`, `unstable__tokio`, `unstable__async-std` or `unstable__async_std`

Reply:
doesn't look like depends on any imports in `std`, does it